service-workers/service-worker/resources/no-dynamic-import.js

```diff
@@ -0,0 +1,22 @@
+/** @type {[name: string, url: string][]} */
+const importUrlTests = [
+  ["Module URL", "./resources/basic-module.js"],
```

Comment:
Shouldn't this be relative to the directory of the file? I thought that's how imports did it. (Same below.)

Reply:
Oh good catch. This was using `new URL` to do the relative stuff and I botched the refactor.